.circleci/config.yml

```diff
@@ -257,7 +257,52 @@ jobs:
           root: .
           paths:
             - ui
-
+  
+  puppeteer-e2e-test:
+    docker:
+      - image: circleci/node:12.16.1
+    working_directory: ~/workbench
+    steps:
+      - checkout
+      - run:
+          name: Fetch Submodules
+          command: git submodule update --init --recursive
+      - restore_cache:
+          keys:
+          - ui-cache-{{ checksum "~/workbench/e2e/yarn.lock" }}
+          - ui-cache-
+      - run: 
+          working_directory: ~/workbench/e2e
+          command: yarn cache clean && yarn install --verbose --frozen-lockfile
+      - save_cache:
+          paths:
+            - ~/.cache/yarn
+          key: ui-cache-{{ checksum "~/workbench/e2e/yarn.lock" }}
+      - run:
+          working_directory: ~/workbench/e2e
+          name: Install Headless Chrome dependencies
+          command: |
+            sudo apt-get install -yq \
```

Comment:
Can we bake some of these into the dockerfile? This seems like it would be a large amount of files to install every time.

Reply:
I'd be surprised if there isn't a prebuilt Circle image which has these. Try circleci/node:12.16.1-browsers

Reply:
Thanks, this is good. We just don't want to maintain this long list of package installs if we can avoid it. Image size of prebuilt Circle images is not really a concern since Circle automatically caches them all.